lib/Models/Catalog/Esri/ArcGisFeatureServerCatalogItem.ts

```diff
@@ -257,7 +256,9 @@ class FeatureServerStratum extends LoadableStratum(
         return undefined;
       }
 
+      // @ts-expect-error @types/proj4 doesn't define a constructor type
       const source = new proj4.Proj((proj4definitions as any)[wkid]);
+      // @ts-expect-error @types/proj4 doesn't define a constructor type
```

Comment:
Are you sure about this? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a7b4fad09a7a1309d0fa9a1ec22c57720bf5d02a/types/proj4/index.d.ts#L71 and https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a7b4fad09a7a1309d0fa9a1ec22c57720bf5d02a/types/proj4/proj4-tests.ts#L65 looks like it should work?

Reply:
`proj4.Proj()` is available but not `new proj4.Proj()`.

I have now modified the code to use only `proj`. Tested by [this spec](https://github.com/TerriaJS/terriajs/blob/baca8d1439eaec1ee46c416399d5f3b7e29e009d/test/Models/Catalog/esri/ArcGisFeatureServerCatalogItemSpec.ts#L109).